compiler/qsc_circuit/src/circuit.rs

```diff
@@ -335,7 +360,15 @@ impl Display for Circuit {
 
             register_to_row.insert((q.id, None), rows.len() - 1);
 
-            for i in 0..q.num_children {
+            // If this qubit has no children, but it is in a multi-qubit operation with
+            // the next qubit, we add one classical row for clarity.
```

Comment:
```suggestion
            // the next qubit, we add an empty row to make room for the vertical connector.
            // We can just use a classical wire type for this row since the wire won't actually be rendered.
```

I now see how you're exploiting the "classical" wire type to create an empty row. I think that's fine (no need to overengineer it and add a 3rd wire type) but it's subtle enough that I think it warrants an explanatory comment.